pkg/actions/lua/lakefs/catalogexport/table_extractor.lua

```diff
@@ -0,0 +1,52 @@
+local pathlib = require("path")
+local strings = require("strings")
+local yaml = require("encoding/yaml")
+local common = require("lakefs/catalogexport/common")
+local hive = require("lakefs/catalogexport/hive")
+
+local LAKEFS_TABLES_BASE = "_lakefs_tables/"
+
+function _is_table_obj(entry, tables_base)
+    if entry == nil or entry.path_type ~= "object" then
+        return false
+    end
+    -- remove _lakefs_tables/ from path 
+    local suffix = entry.path:sub(#tables_base, #entry.path)
```

Comment:
don't think suffix reflects what we extract there - we got the path under lakefs tables.
if this function is for general use - we need to match the prefix before we remove it.

Reply:
Added checked for prefix match and changed a bit.